src/Sulu/Bundle/ContentBundle/Resources/phpcr-migrations/Version201504271608.php

```diff
@@ -53,8 +56,35 @@ private function migrateHome(SessionInterface $session, $from, $to, $referenceWe
             }
 
             $homeNode = $webspace->getNode($homeNodeName);
-            $homeNode->addMixin($to);
+            $tmpNode = $session->getRootNode()->addNode('/tmp');
+            $tmpNode->addMixin('mix:referenceable');
+
+            $session->save();
+
+            $homeNodeReferences = $homeNode->getReferences();
+            $homeNodeReferenceValues = array();
+            foreach ($homeNodeReferences as $homeNodeReference) {
+                /** @var Property $homeNodeReference */
+                $homeNodeReferenceValues[$homeNodeReference->getPath()] = $homeNodeReference->getValue();
+                $homeNodeReference->setValue($tmpNode);
+            }
+
+            $session->save();
             $homeNode->removeMixin($from);
+            $session->save();
+            $homeNode->addMixin($to);
+            $session->save();
```

Comment:
It require me so many session saves in order to make this working... There are a lot of errors I couldn't avoid otherwise (e.g. it seems like the `removeMixin` always removes all mixins, couldn't find out why exactly that happened)